qiskit/transpiler/coupling.py

```diff
@@ -299,14 +299,14 @@ def from_full(cls, num_qubits, bidirectional=True):
 
     @classmethod
     def from_line(cls, num_qubits, bidirectional=True):
-        """Return a fully connected coupling map on n qubits."""
+        """Return qubits connected in a line on n qubits."""
```

Comment:
We should probably start with something like "Return a coupling map of ..." rather than "Return qubits", because `Qubit` is also a class in Qiskit, and we don't want to give the wrong impression.

Reply:
Clear. Hopefully further clarified by adding a return type in a separate commit (can revert if desired)